modules/repository/repo.go

```diff
@@ -70,6 +72,75 @@ func MigrateRepositoryGitData(ctx context.Context, u *models.User, repo *models.
 		return repo, fmt.Errorf("Clone: %v", err)
 	}
 
+	if opts.LFS {
```

Comment:
We have to check if `git lfs` is installed when Gitea start or we can just use git-lfs library since it's also a Golang library.